kura/org.eclipse.kura.core.keystore/OSGI-INF/metatype/org.eclipse.kura.core.keystore.FilesystemKeystoreServiceImpl.xml

```diff
@@ -45,8 +45,88 @@
             description="Specifies if the defined password will be randomized at the next keystore access. If this value is set to true and the keystore can be accessed, the password will be randomized and this field will automatically set to false.">
         </AD>
 
+        <AD id="crl.management.enabled"
+            name="CRL Cache Enabled"
+            type="Boolean"
+            cardinality="0"
+            required="true"
+            default="false"
+            description="Enables or disables CRL cache.">
+        </AD>
+
+        <AD id="crl.update.interval"
+            name="CRL Force Update Interval"
+            type="Long"
+            cardinality="0"
+            required="true"
+            default="1"
+            description="Defines a time interval for forcing the update of cached CRLs, this interval will be considered in addition to CRL next update date.">
```

Comment:
Does this work only if a URL is set in `crl.urls`?

Reply:
No, the HTTP distribution points specified in the trusted certificates in the keystore are considered as well, in addition to the URLs specified in configuration.